### ISSUE TITLE
Toggle menubar option

### ISSUE THREAD

**Issue by RixzZ** [notable/notable#546]
### Feature description

Option to hide the menu bar. Maybe be able to show it briefly when pressing Alt as lots of other apps do.

### Feature motivation

You can gain a little vertical space and the app will look a bit better in my opinion. Also it currently breaks the simplicity of Zen or Focus mode.

This app also have all needed options on the main interface or shortcuts for them, so there is no need to have the menubar permanently shown.

**Comment by fabiospampinato** [comment]
It actually worked like that in the earlier releases, but many people weren't aware of this pattern and struggled to find the menu: https://github.com/notable/notable/issues/138 https://github.com/notable/notable/issues/183 https://github.com/notable/notable/issues/3

So this isn't coming back, but there are some plans to implement custom titlebar + menubar for Linux and Windows too #315, then the menubar will be much less intrusive.

I agree that right now it kind of butchers the design a bit.

**Comment by Philiphlop** [comment]
If you've already created the code, why not make it optional? Have it on by default, but then user can switch to hide by default. Just alongside the others

`View > Hide Menu Bar                   [Alt]`

The experience is the same with Full Screen, there is no menu there, and to get back out you have to know to press F11 as is shown in the menu list. 

I am always in favour of user configurable items like this, as everyone is a bit different, so tweaks can make it perfect for most. Especially with things like this which are not heavy changes, prone to bugs, and as I understand it just a config that Notable has to pass to Electron.

**Comment by fabiospampinato** [comment]
@Philiphlop sounds like a good idea. I didn't really wrote any actual logic for this, it was more like I toggled a flag in a configuration object. And I think it would be better to go the opposite route and display it by default, but add the option to hide it, as people have had troubled finding the menubar in the past.

**Comment by davistran86** [comment]
I hope that I can hide the menu bar and even the title bar as well :)

**Comment by jasonwhat** [comment]
This may be a bug, but I'm on the latest version of Notable 5.1, Linux on ChromeOS and Alt does not make the menu appear. I can't figure out any combo that makes it appear actually. Am I missing something?

**Comment by fabiospampinato** [comment]
@jasonwhat it should be shown by default, which doesn't sound like it's actually the case for you. I'm not very familiar with ChromeOS, can you post a screenshot of the app running there? And what packaging format did you use, `deb`?

**Comment by jasonwhat** [comment]
I used deb and the built in installer on ChromeOS/Linux.

I use Typora and "alt" makes the menubar appear with no issue.

![Screenshot 2019-05-15 at 1 51 47 PM](https://user-images.githubusercontent.com/4491030/57797624-d4864300-7718-11e9-91cb-467b782fa19f.png)

**Comment by jasonwhat** [comment]
I've tried to remove the package and reinstall with no changes. I also tried "purge" to see if I could get rid of any configuration, but when I open Notable it picks up where I left off -- there's no tutorial or prompt to pick a directory. 

Is there some command I can run to reset Notable and get it to treat me as a first time user?

Out of curiosity, I tried installing the snap version (after installing snap), but that seems to have dependency issues.
